fluent-langneg/src/accepted_languages.js

```diff
@@ -1,7 +1,24 @@
-export default function acceptedLanguages(string = "") {
-  if (typeof string !== "string") {
+export default function acceptedLanguages(acceptLanguageHeader = "") {
+  if (typeof acceptLanguageHeader !== "string") {
     throw new TypeError("Argument must be a string");
   }
-  const tokens = string.split(",").map(t => t.trim());
-  return tokens.filter(t => t !== "").map(t => t.split(";")[0]);
+  const tokens = acceptLanguageHeader.split(",").map(t => t.trim());
+  const langsWithQ = [];
+  tokens.filter(t => t !== "").forEach((t, index) => {
+    const langWithQ = t.split(";").map(u => u.trim());
+    if (langWithQ[0].length > 0) {
+      let q = 1.0;
+      if (langWithQ.length > 1) {
+        const qVal = langWithQ[1].split("=").map(u => u.trim());
+        if (qVal.length === 2 && qVal[0].toLowerCase() === "q") {
+          const qn = Number(qVal[1]);
+          q = !isNaN(qn) ? qn : q;
```

Comment:
That means that any non-numerical `q` value will get replaced with `1.0`, is that what we should do? Seems like bogus input should result in low priority, not high.